ingestion/functions/parsing/ch_zurich/input_event.json

```diff
@@ -1,5 +1,6 @@
 {
     "s3Bucket": "epid-sources-raw",
+    "sourceId": "5f2acbe8df34a7051c8b01f4",
```

Comment:
hyper-nit, but can we put this after the s3 params throughout (i.e. here and in the retrieval file method signature below)?

Reply:
I changed here but in the python code it feels better close next to the sourceURL as it is now rather than having source url,s3 bucket,s3 key,source id